src/calculator.js

```diff
@@ -16,7 +16,7 @@ exports.add = (x, y) => {
   return x + y;
 };
 
-exports.subtract = (x, y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule